### DISCUSSION TITLE
A few suggestions

### DISCUSSION THREAD

**Discussion by KidFast** [ppy/osu#22542] [Ideas]
Hello! I am really excited for osu! lazer's final release, it's looking like an amazing project so far. I do have some suggestions that will definitely make the game outshine it's predecessor. I did not read through other suggestions so pardon me if I repeat anyone's idea. I am not yelling where I use all caps. sorry
Suggestions:
1. LET THERE BE PRESET SKINS FOR SPECIFIC MANIA KEY MODES: I would like skin presets, I noticed there we're mod presets you could make, it would be nice to have a preset skin for specific key modes. I use circle skins for 4 key mode on osu mania and bar skins for 7key mode. Sometimes when i'm playing multiplayer I forget to switch skins. This should Help!
2. A MORE DETAILED CHART FOR HITS: games like quaver and etterna have very detailed charts or informative features that provide players with necessary info on where their hits lie mid game, the UR bar is fine and helps, but a more detailed version is hopefully not too hard to implement and is not an opportunity that should be missed on.

before submitting I noticed I'm meant to search for similar discussions and then did
for my first suggestiont he closest was someone asking for skin presets for specific mods. which is also a great idea

**Comment by Rekunan** [comment]
For 2 do you mean the hit scatter plot at the bottom right here?
![Etienne - (Etterna) me   u )  DGWQ4Eubu6g - 1920x1080 - 3m09s](https://user-images.githubusercontent.com/24501632/217418985-2b77e874-cbe5-4c66-aa6e-50e0dac6fe02.png)

**Reply by KidFast** [reply]
Yes exactly what I mean, thank you for the helpful image, maybe players could toggle it having a simplified graph and a complex one so not to confuse any new players
